### PR TITLE
Add borrowings create endpoint

### PR DESCRIPTION
**Pull request by CaCtUs1k**

### CODE REVIEW

File: borrowings/serializers.py [hunk 1]
```diff
@@ -19,5 +21,45 @@ class Meta:
...
+    class Meta:
+        model = Borrowing
+        fields = ("id", "book", "expected_return_date", "actual_return_date")
+
+    def validate(self, attrs):
+        data = super().validate(attrs)
+
+        print(data)
+        book_pk = data.get("book").pk
+        print(book_pk)
```
Comment (Barsh4ec) [inline borrowings/serializers.py:44]: remove prints here

**Review by Barsh4ec — CHANGES_REQUESTED** [review]

File: borrowings/serializers.py (hunk 1 above)
Comment (CaCtUs1k) [reply borrowings/serializers.py:44]: xd

**Review by Barsh4ec — APPROVED** [review]
lgtm

File: borrowings/tests/test_borrowing_view.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+from books.models import Book
+from borrowings.models import Borrowing
+
+
+class BorrowingTests(TestCase):
+
+    def setUp(self):
+        self.client = APIClient()
+        self.user = get_user_model().objects.create_user(email="test_user@test.com", password="testpassword")
```
Comment (ochernous) [inline borrowings/tests/test_borrowing_view.py:17]: long lines

File: borrowings/views.py [hunk 3]
```diff
@@ -10,5 +13,15 @@ class BorrowingViewSet(viewsets.ModelViewSet):
...
         if self.action == "retrieve":
             return BorrowingDetailSerializer
+        if self.action == "create":
+            return BorrowingCreateSerializer
         else:
             return BorrowingSerializer
+
+    def perform_create(self, serializer):
+        with transaction.atomic():
+            book_pk = serializer.validated_data['book'].pk
```
Comment (ochernous) [inline borrowings/views.py:23]: quotes

**Review by eduardhabryd — APPROVED** [review]
Good Job!

**Pull request merged**